src/main/java/core/basesyntax/services/impl/FruitReportServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.services.FruitReportService;
+import core.basesyntax.services.TransactionParser;
+import core.basesyntax.services.Validator;
+import core.basesyntax.strategy.Operation;
+import java.util.List;
+import java.util.Map;
+
+public class FruitReportServiceImpl implements FruitReportService {
+    public static final String COMMA = ",";
+
+    @Override
+    public String createReport(List<String> listData,
+                               Map<Transaction.Operation, Operation> operationMap) {
+        Validator validator = new ValidatorImpl();
+        TransactionParser parserService = new TransactionParserImpl();
```

Comment:
Services we rely on shouldn't be initialized with 'new' in method.
With such practice we create new object each time method is called and making code very hard to maintain and test
Please make services class variable and initialize them via constructor